umake/frameworks/baseinstaller.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, *args, **kwargs):
         self.desktop_filename = kwargs.get("desktop_filename", None)
         self.icon_filename = kwargs.get("icon_filename", None)
         self.match_last_link = kwargs.get("match_last_link", False)
+        self.exec_path = kwargs.get("exec_path", False)
```

Comment:
So, we can maybe change this for:
`self.exec_rel_path=`
Set that to the relative path (relative to installation path), and change that in each __init__() to add this key.

Then, we set self.exec_path base on self.install_path and self.exec_rel_path in `confirm_path()`

That way, you can remove in post_install the duplication of setting the self.exec_path, which can be error prone

Making sense?

PS: do not forget to change the tests/large/test_baseinstaller.py as well to take that one into account ;) (with and without a desktop filename)